remove redundant example for cat, update snapshot

kdiab opened the pull request:
This PR is for issue  #1186

`cat` was documented twice causing it to appear twice in the API reference. This change removes the less descriptive example and keeps the other one. 

![image](https://github.com/user-attachments/assets/705d4dc9-373e-4808-856e-607ff6df8d56)

felixroos commented:
thanks for your contribution. It looks like this example you've removed is still referenced (as Pattern.cat) in https://strudel.cc/learn/factories/#cat , so maybe add it there as a MiniRepl instead?

kdiab commented:
Yeah I think that's a good idea! This looks more informative.

![image](https://github.com/user-attachments/assets/6d6b0b0b-b1c5-4d53-aa8b-e244fb01b882)

kdiab commented:
Found the same bug for `seq` and `stack`, going to combine with this PR. 
![image](https://github.com/user-attachments/assets/de74483f-4949-43b0-8bd3-de122b016970)
![image](https://github.com/user-attachments/assets/71316e0a-f57b-4023-b77b-f15170d5d7d7)

felixroos commented:
>  It looks like this example you've removed is still referenced (as Pattern.cat) in https://strudel.cc/learn/factories/#cat 

have you looked at this?  seq and stack might have refs as well

kdiab commented:
Yes! That's where I got the examples from 😊

felixroos commented:
i mean have you checked if the factories page still looks as before? this: https://github.com/tidalcycles/strudel/blob/e409086b34c9a226e518a190ed86cc2510c640be/website/src/pages/learn/factories.mdx?plain=1#L38 just to double check

kdiab commented:
Removed unnecessary instances of functions and fixed the factories folder.

![image](https://github.com/user-attachments/assets/a55d734d-33bd-4c49-8cfd-2441f96314e7)
![image](https://github.com/user-attachments/assets/82823c48-402a-4c73-8bba-4fb123e8bfd1)
![image](https://github.com/user-attachments/assets/26c71af8-606e-40ad-b022-277f6aa17312)
![image](https://github.com/user-attachments/assets/33014294-5f42-4157-9da9-9cf4315b08f6)
![image](https://github.com/user-attachments/assets/c245f001-5a74-4319-9ef9-25116e479116)

felixroos commented:
thanks, this now looks good!
